crates/e2e/tests/e2e/onchain_settlement.rs

```diff
@@ -78,8 +81,21 @@ async fn onchain_settlement(web3: Web3) {
     );
 
     let services = Services::new(onchain.contracts()).await;
-    services.start_autopilot(vec![]);
-    services.start_api(vec![]).await;
+    let solver_endpoint = colocation::start_solver(onchain.contracts().weth.address()).await;
+    colocation::start_driver(
+        onchain.contracts(),
+        vec![SolverEngine {
```

Comment:
Maybe we need to start the naive solver here for it to pick up the CoW.

Reply:
That helped, thanks 👍